cross-platform/react-app/src/frontend/Screens/Hub/ProjectPicker.tsx

```diff
@@ -4,42 +4,70 @@
 *--------------------------------------------------------------------------------------------*/
 import React from "react";
 import { HorizontalPicker, useIsMountedRef } from "@itwin/mobile-ui-react";
-import { Project, ProjectsAccessClient, ProjectsQueryArg, ProjectsQueryFunction, ProjectsSearchableProperty, ProjectsSource } from "@itwin/projects-client";
+import { ITwin, ITwinsAccessClient, ITwinsAPIResponse, ITwinsQueryArg, ITwinSubClass } from "@itwin/itwins-client";
 import { IModelApp } from "@itwin/core-frontend";
 import { LoadingSpinner } from "@itwin/core-react";
 import { ButtonProps, HubScreenButton, HubScreenButtonList, HubScreenButtonListProps, i18n, presentError, PromiseUtil, SearchControl, useLocalizedString } from "../../Exports";
 
+type WithRequired<T, K extends keyof T> = T & { [P in K]-?: T[P] };
+export type Project = WithRequired<ITwin, "id">;
+
+enum ProjectsSource {
+  All = "",
+  Favorites = "favorites",
+  Recents = "recents"
+}
+
+interface ProjectsQueryResponse {
```

Comment:
Because this is referenced from another documentation comment, it should also be documented.

Reply:
Fixed